src/state_client.h

```diff
@@ -0,0 +1,120 @@
+#ifndef AUTOSAR_STATE_MANAGEMENT_STATE_CLIENT_H
+#define AUTOSAR_STATE_MANAGEMENT_STATE_CLIENT_H
+/** @file state_client.h
+ * @brief State Client EM interface. */
+
+#include "sm_types.h"
+
+namespace ara { namespace exec {
```

Comment:
please merge namespaces as we are using cpp17

Reply:
Done